framework/commercelayer/next.config.js

```diff
@@ -0,0 +1,8 @@
+const commerce = require('./commerce.config.json')
+
+module.exports = {
+  commerce,
+  images: {
+    domains: ['localhost', 'data.commercelayer.app'],
```

Comment:
```suggestion
    domains: ['data.commercelayer.app'],
```
localhost is not needed for domains